### PR TITLE
add warning in lims docs about values outside of limits being removed from data

### LINKED ISSUE #3752: Improve lims documentation
lims documentation should add a clear warning about using scale limits for zooming because it removes data and can lead to artefacts.

### PR DESCRIPTION
**Pull request by Ryo-N7**
improve lims docs: closes #3752

### CODE REVIEW

File: R/limits.r [hunk 1]
```diff
@@ -17,10 +17,13 @@
 #'  A character or factor value will create a discrete scale.
 #'  A date-time value will create a continuous date/time scale.
 #'
-#' @seealso For changing x or y axis limits \strong{without} dropping data
-#'   observations, see [coord_cartesian()]. To expand the range of
-#'   a plot to always include certain values, see [expand_limits()]. For other
-#'   types of data, see [scale_x_discrete()], [scale_x_continuous()], [scale_x_date()].
+#' @seealso As any values outside the limits specified are replaced with `NA`,
+#'   be warned that this will result in your data outside the limits being
+#'   removed and this can produce unintended results. For changing x or y axis
```
Comment (clauswilke) [inline R/limits.r]: I don't think the text you added belongs into the `@seealso` section. I think it's better to expand the very first paragraph, where it already talks about data points being replaced with `NA`.

**Comment by thomasp85** [comment]
@clauswilke can you finish this off with @Ryo-N7 ?

**Comment by clauswilke** [comment]
@Ryo-N7 I'm fine with the revised text. Could you rebuild the documentation so the .Rd file matches the .R file?

**Comment by Ryo-N7** [comment]
done!

**Comment by clauswilke** [comment]
Thanks!

**Pull request merged**